From 190b8bbe7d94fe72636380d20585a0c7a6aec54c Mon Sep 17 00:00:00 2001
From: Pedro Palazon Candel <pedro@joyent.com>
Date: Tue, 25 Sep 2018 15:55:04 +0200
Subject: [PATCH] TRITON-785 Add requirements.brand and requirements.bootrom to
 AdminUI's images view Reviewed by: Mike Gerdts <mike.gerdts@joyent.com>
 Reviewed by: Marsell Kukuljevic <marsell@joyent.com> Approved by: Marsell
 Kukuljevic <marsell@joyent.com>

---
 www/js/tpl/image.hbs | 16 +++++++++++++++-
 1 file changed, 15 insertions(+), 1 deletion(-)

diff --git a/www/js/tpl/image.hbs b/www/js/tpl/image.hbs
index e5bb027f..449fa510 100644
--- a/www/js/tpl/image.hbs
+++ b/www/js/tpl/image.hbs
@@ -5,7 +5,7 @@
 -->
 
 <!--
-    Copyright (c) 2014, Joyent, Inc.
+    Copyright (c) 2018, Joyent, Inc.
 -->
 
 <div class="page-header">
@@ -240,6 +240,20 @@
         </span>
       </li>
       {{/if}}
+
+      {{#if requirements.bootrom}}
+      <li>
+        <strong>BootROM</strong>
+        <span class="value">{{requirements.bootrom}}</span>
+      </li>
+      {{/if}}
+
+      {{#if requirements.brand}}
+      <li>
+        <strong>Brand</strong>
+        <span class="value">{{requirements.brand}}</span>
+      </li>
+      {{/if}}
     </ul>
     </section>
 
-- 
2.21.0

